### ISSUE TITLE
Flag `if (A) { B } else if (A) { C }`

### ISSUE THREAD

**Issue by filiph** [dart-lang/linter#177]
``` dart
  if (aSource.mType == NS_STYLE_CLIP_PATH_URL) {
    SetURL(aSource.mURL);
  } else if (aSource.mType == NS_STYLE_CLIP_PATH_SHAPE) {
    SetBasicShape(aSource.mBasicShape, aSource.mSizingBox);
  } else if (aSource.mType == NS_STYLE_CLIP_PATH_SHAPE) {
    SetSizingBox(aSource.mSizingBox);
  }
```

Seems to be another common copy-paste error.

**Comment by bwilkerson** [comment]
@alexeieleusis Can you confirm that this case is covered?

**Comment by alexeieleusis** [comment]
It is linted with the specified rule, tested with the following snippet:

```
void nestedOk4_1() {
  if (foo <= bar) {
    return;
  } else if (foo <= bar) { // LINT
    print('hola mundo!');
  } else if (foo <= bar) { // LINT
    print('hello world!');
  }
}
```
Should we add it as a test case?

**Comment by bwilkerson** [comment]
If there isn't already a test to test this case, that would be a good idea, but there's no point in testing it twice.
